client/js/components/day/staff-card.js

```diff
@@ -0,0 +1,43 @@
+var React = require('react');
+var _ = require('underscore');
+var StaffGroup = require('./staff-group');
+
+var Staff = React.createClass({
```

Comment:
The variable name of staff here isn't ideal. For one, the filename is staff-card so we'd want those aligned. Also, this represents the event bin or event bucket. When we implement drag and drop of employees, if you pick one employee up and move them to another bucket, the specific change that's being made is the employee is working at a different event. It's less meaningful to say they are now working on a different staff.

As part of our Spring cleaning, I think we should cleanup our naming conventions and solidify the difference between a Kanban board, bin, and card in our application.